packages/cli/src/index.ts

```diff
@@ -223,7 +223,11 @@ applyCommandsConfig(program.command('build'), commandsConfig.build)
     const pickedCliSettings = _.pick(cliSettings, Object.keys(options));
     const mergedOptions = _.assign({}, options, pickedCliSettings);
 
-    const [node, pkgSpec, outputs, runtime] = await doBuild(cannonfile, settings, mergedOptions);
+    const [node, pkgSpec, outputs, runtime, deployInfo] = await doBuild(cannonfile, settings, mergedOptions);
+
+    if (deployInfo.status !== 'complete') {
+      process.exitCode = deployInfo.status === 'partial' ? 10 : 11;
```

Comment:
Do we want 10 and 11 here? Google says 11 is `segmentation fault error` ?

Reply:
You are right, is more common to use higher numbers for custom errors, updating to 90 🙏